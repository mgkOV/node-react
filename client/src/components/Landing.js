import React from 'react';

const Landing = () =>
  <div style={{ textAlign: 'center' }}>
    <h1>Emaily</h1>
    <div>Collect feedback form your users</div>
  </div>;

export default Landing;
