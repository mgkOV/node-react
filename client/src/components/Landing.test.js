import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

describe('Landing tests', () => {
  test('Landing renders correctly', () => {
    const component = shallow(<Landing />);
    expect(component).toMatchSnapshot();
  });
});
