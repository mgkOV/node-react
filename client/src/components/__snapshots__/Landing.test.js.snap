// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing tests Landing renders correctly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Emaily
  </h1>
  <div>
    Collect feedback form your users
  </div>
</div>
`;
